vue-project/src/components/PrintJobs.vue

```diff
@@ -149,47 +113,36 @@ const validateCreatePrintSettings = () => {
   return true;
 }
 
-//// API CALLS ////
-const createPrintSettings = async () => {
-  if (!validateCreatePrintSettings()) {
-    return false;
-  }
-
-  const url = "http://54.200.253.84:80/createJob";
-  const data = {
-    Title: printSettings.value.title.toString(),
-    PageCount: printSettings.value.pageCount.toString(),
-    RasterizationProfile: printSettings.value.rasterizationProfile.toString()
-  }
-
-  const response = await fetch(url, {
-    method: 'POST',
-    mode: 'cors',
-    headers: { 'Content-Type': 'application/json' },
-    body: JSON.stringify(data)
-  });
+  //// API CALLS ////
+  const createPrintSettings = async () => {
+    if (!validateCreatePrintSettings()){
+      return false;
+    }   
+
+    const url = "http://54.200.253.84:80/createJob";
```

Comment:
Might want to use `api.wsuv-hp-capstone.com` instead of IP, but we can change that later